lib/types.go

```diff
@@ -41,6 +42,18 @@ type Options struct {
 	Thresholds map[string][]*Threshold `json:"thresholds"`
 }
 
+type SourceData struct {
```

Comment:
Like, say, a type for JSON files, plain text files, etc.?

I think that would belong here, but it should be separate from the runner type, with different constants - script, datastore, plaintext, etc. - and added when we have a use for it.